testing/test-scripts/demo_script.sh

```diff
@@ -14,30 +14,74 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-
 set -x
 set -o errexit
 
+# Set env vars
+export A8_CONTROLLER_URL=http://localhost:31200
+export A8_REGISTRY_URL=http://localhost:31300
+export A8_GATEWAY_URL=http://localhost:32000
+export A8_LOG_SERVER=http://localhost:30200
+export A8_GREMLIN_URL=http://localhost:31500
+
+if [ -z "$A8_TEST_GREMLIN" ]; then
+    A8_TEST_GREMLIN="true"
+fi
+
 jsondiff() {
     # Sort JSON fields, or fallback to original text
     file1=$(jq -S . $1 2> /dev/null) || file1=$(cat $1)
     file2=$(jq -S . $2 2> /dev/null) || file2=$(cat $2)
-    
+
     # Diff, but ignore all whitespace since
-    diff -Ewb <(echo $file1) <(echo $file2)        
+    diff -Ewb <(echo $file1) <(echo $file2)
 }
 
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+make -C $SCRIPTDIR/../../ build.cli
+CLIBIN=$SCRIPTDIR/../../bin/a8ctl-beta
```

Comment:
On a second thought, I think it would be better if we build the CLI in the makefile (as a dependency of test.integration) otherwise users will have to build the CLI manually before running the integ test locally.